src/services/ServerIO/RoomIO.js

```diff
@@ -2,12 +2,14 @@ class RoomIO {
     constructor(setup, serverIO) {
         const {
             id,
+            name,
             isPrivate = false,
             participants = []
         } = Object(setup);
 
         this._serverIO = () => serverIO;
         this.id = id;
+        this.name = name || id;
```

Comment:
This is something that need to be revised, because I added to fix a specific bug on the test project **chat-app**